src/uwp/build.proj

```diff
@@ -34,4 +34,8 @@
     <Copy SourceFiles="@(CMakeOutput)"
           DestinationFolder="$(UWPOutputDir)"/>
   </Target>
+
+  <Target Name="BuildMSBuildTask" DependsOnTargets="GetLatestCommitHash">
+    <MSBuild Projects="$(MSBuildThisFileDirectory)Microsoft.Build.Net.CoreRuntimeTask/Microsoft.Build.Net.CoreRuntimeTask.builds" />
```

Comment:
Given that it is a .builds file, should we build this node in parallel? (pass in `/m`)